infrastructure/core/main.tf

```diff
@@ -100,17 +100,27 @@ resource "azurerm_log_analytics_workspace" "core" {
   resource_group_name        = azurerm_resource_group.core.name
   internet_ingestion_enabled = var.tf_in_automation ? false : true
   sku                        = "PerGB2018"
-  retention_in_days          = 30
+  retention_in_days          = 90
+  internet_query_enabled     = false
```

Comment:
worth discussing: (1) do we want to disable this and (2) without an admin jumpbox how does how is it possible to access logs?

Reply:
I think this should be left enabled, would make monitoring very difficult. @damoodamoo thoughts?

Reply:
discussed on slack – will disable in prod with a manual step to enable